tests/test_node.py

```diff
@@ -215,3 +221,35 @@ def test_node_move_across_games(self):
                           self.game.root, self.extensive_game.root)
         self.assertRaises(pygambit.MismatchError, self.game.move_tree,
                           self.extensive_game.root, self.game.root)
+
```

Comment:
It would make more sense to have all of the `append_move` and `append_infoset` tests together in one place.  For the moment `test_node` is the better place (if only because it's already been converted to `pytest` format).